snapshots/dummy/2020-01-01/dummy.csv.dvc

```diff
@@ -1,17 +1,13 @@
 meta:
-  namespace: dummy
-  short_name: dummy
   name: Dummy dataset
-  version: '2020-01-01'
   publication_date: '2020-01-01'
   source_name: Dummy short source citation
   source_published_by: Dummy full source citation
   url: https://www.url-dummy.com/
   source_data_url: https://raw.githubusercontent.com/owid/etl/master/walkthrough/dummy_data.csv
-  file_extension: csv
   license_url:
   license_name: ''
-  date_accessed: 2023-05-15
+  date_accessed: 2023-06-20
```

Comment:
Was this changed by accident or did you re-run the snapshot?

Reply:
I'm occasionally using it for testing.